### DISCUSSION TITLE
canva

### DISCUSSION THREAD

**Discussion by arslan1227** [community/community#30275] [GitHub Education]
i dont see a canva option in student developer pack??

**Comment by melvincwng** [comment]
@arslan1227, if I'm not mistaken, Canva is no longer one of the benefits for the GitHub Student Developer Pack.

Please refer to this [link here](https://education.github.com/pack) for a full and updated list of benefits under GitHub Student Developer Pack.

Hope this answers your question! Thanks 😄

**Comment by arslan1227** [comment]
thnx🙂
